pkg/apis/messaging/v1alpha1/kafka_channel_defaults.go

```diff
@@ -33,7 +33,7 @@ func (c *KafkaChannel) SetDefaults(ctx context.Context) {
 		c.Annotations = make(map[string]string)
 	}
 	if _, ok := c.Annotations[messaging.SubscribableDuckVersionAnnotation]; !ok {
-		c.Annotations[messaging.SubscribableDuckVersionAnnotation] = "v1alpha1"
+		c.Annotations[messaging.SubscribableDuckVersionAnnotation] = "v1beta1"
```

Comment:
I think this will break things.

This annotation states what Subscribable duck version this type is using.

KafkaChannel v1alpha1 is still using v1alpha1 duck version.

Reply:
I was looking at this: https://github.com/knative/eventing/pull/3169/files

The storage version doesn't shouldn't break the actual code, because we have the type conversions... Right? At least, that's what I understood

Reply:
ok, then do we change KafkaChannel type as well?

Reply:
Seems like all the integration tests passed :smile:

Reply:
Ok, cool!

Reply:
@vaikas can you double check this? If it's wrong, we revert